_posts/2017-12-27-cheap-docker-swarm-cluster-aws-terraform.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title:  "Build a Cheap Docker Swarm Cluster using Terraform and AWS"
+date:   2017-12-25 12:00:00
+author:
+  display_name: Ryan Sadler
+  login: sadler
+  email: sadler@mysticcoders.com
+  url: https://www.mysticcoders.com
+comments: true
+---
+![screenshot](/images/2017-12-27-docker-horizontal-large.png)
+
+I set out to build  a super cheap Swarm Cluster for a personal project.  With a little help from terraform you can be up and running with a multi-node swarm cluster in a matter of minutes. We aren’t going to get into what you can do with your cluster yet, that will be a future article.  We’re just going to look at prices and see how cheap we can do this.
+
+<!--more-->
+
+## Overview
+
+Why use Swarm, when you could just spin up some cheap servers?  The nice thing about swarm is you get a lot of cluster functionality out of the box, not to mention that it will run your docker containers. Things like Service Discovery and Rolling updates are easy with Swarm across the entire cluster.  If you’re building an app that needs to be updated periodically, this is a huge help.  [See more about Swarm](https://docs.docker.com/engine/swarm/)
```

Comment:
TF is swarm? Does your reading audience know this already? (Hint: assume not. You don't have to write a full tutorial, just SUMMARIZE.)

And no, "read more about swarm" at the end of the paragraph isn't good enough. This is a web page. Use the web.